docs/rules/capitalized-comments.md

```diff
@@ -0,0 +1,241 @@
+# enforce or disallow capitalization of the first letter of a comment (capitalized-comments)
+
+This rule was ported from JSCS' [requireCapitalizedComments](http://jscs.info/rule/requireCapitalizedComments) and [disallowCapitalizedComments](http://jscs.info/rule/disallowCapitalizedComments) rules.
+
+
+## Rule Details
+
+This rule aims to enforce a consistent style of comments across your codebase, specifically by either requiring or disallowing a capitalized letter as the first word character in a comment. This rule will not issue warnings when non-cased letters are used.
+
+By default, this rule will require a non-lowercase letter at the beginning of comments.
+
+Examples of **incorrect** code for this rule:
+
+```js
+/* eslint capitalized-comments: ["error"] */
+
+// lowercase comment
+
+```
+
+Examples of **correct** code for this rule:
+
+```js
+
+// Capitalized comment
+
+// 1. Non-letter at beginning of comment
+
+// 丈 Non-Latin character at beginning of comment
+
+/* eslint semi:off */
+/* eslint-env node */
+/* eslint-disable */
+/* eslint-enable */
+/* istanbul ignore next */
+/* jscs:enable */
+/* jshint asi:true */
+/* global foo */
+/* globals foo */
+/* exported myVar */
+// eslint-disable-line
+// eslint-disable-next-line
+// https://github.com
+
+```
+
+### Options
+
+This rule has one option: either a string value `"always"` or `"never"` for simple configuration, or an object containing all possible configuration parameters for the rule.
+
+If you want finer control of this rule, you must supply an options object in your configuration file.
+
+Here are the supported options:
+
+* `capitalize`: This represents whether comments are expected to start with a capitalized letter. Available values are `"always"` and `"never"`, with the default being `"always"`. (The string-only options correspond to this option object property.)
+* `ignorePattern`: A string representing a regular expression pattern of words that should be ignored by this rule. If the first word of a comment matches the pattern, this rule will not report that comment.
+    * Note that the following words are always ignored by this rule: `["jscs", "jshint", "eslint", "istanbul", "global", "globals", "exported"]`.
+* `ignoreInlineComments`: If this is `true`, the rule will not report on comments in the middle of code. By default, this is `false`.
+* `tolerateConsecutiveComments`: If this is `true`, the rule will not report on a comment which violates the rule, as long as the comment immediately follows a comment which is also not reported.
```

Comment:
For sure! Not sure what I was thinking earlier but it's obvious now. I'll go with `ignore` due to fewer characters.